api/models.py

```diff
@@ -64,3 +64,18 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event (models.Model):
+
+	EVENT_STATUS = (
+		('UPCOMING', 'Upcoming events'),
+		('CURRENT', 'Todays events'),
+		('PASSED', 'Passed events'),
+	)
+
+	title = models.CharField(max_length=255, default=None)
+	start_date = models.DateTimeField()
```

Comment:
There is a start date, which implies there should be an end date as well?

Are we supporting multi days events and if so, then we should write down somewhere the end date...

Or we can just not support multi days events.